client/fileutils.c

```diff
@@ -649,6 +649,79 @@ int loadFileDICTIONARY(const char *preferredName, void *data, size_t *datalen, u
     return retval;
 }
 
+int loadFileDICTIONARY_safe(const char *preferredName, uint8_t **data, uint8_t keylen, uint16_t *keycnt) {
```

Comment:
all loadFile* used to use void *data.
Any reason to move to uint8_t (and your numerous internal casts) ?
I'd also rename data into pdata, to show the indirection.
So void **pdata ?

Reply:
Yup that sounds like a good plan ;)